packages/devtools-launchpad/webpack.config.js

```diff
@@ -93,27 +93,29 @@ module.exports = (webpackConfig, envConfig, options = {}) => {
       loader: "svg-inline-loader"
     });
 
+    const cssUses = [{
+      loader: "style-loader"
+    }, {
+      loader: "css-loader",
+      options: {
+        importLoaders: 1,
+        url: false
+      }
+    }];
+
+    if (options.disablePostCSS !== true) {
```

Comment:
NIT, i'm not sure if we have a consensus here, but I'd prefer `!!` when it's either a boolean or undefined. Other cases where the type is overloaded should be more verbose

`if (options.disablePostCSS !== true) {` => `if (!!options.disablePostCSS) {`

Reply:
done